src/arch/x86/smbios.c

```diff
@@ -408,7 +408,7 @@ static int smbios_write_type0(unsigned long *current, int handle)
 	t->handle = handle;
 	t->length = len - 2;
 
-	t->vendor = smbios_add_string(t->eos, "coreboot");
+	t->vendor = smbios_add_string(t->eos, CONFIG_MAINBOARD_SMBIOS_VENDOR);
```

Comment:
```suggestion
	t->vendor = smbios_add_string(t->eos, smbios_mainboard_bios_vendor());
```
otherwise what for we create such function in `src/arch/x86/smbios_defaults.c`?

Reply:
@miczyg1 yeah this is not finished, that's why there is no such call here.